Model/Adminhtml/Source/LpmMethods.php

```diff
@@ -0,0 +1,32 @@
+<?php
+declare(strict_types=1);
+
+namespace Magento\Braintree\Model\Adminhtml\Source;
+
+use Magento\Framework\Data\OptionSourceInterface;
+
+/**
+ * Class LpmMethods
+ */
+class LpmMethods implements OptionSourceInterface
+{
+
+    /**
+     * Return array of options as value-label pairs
+     *
+     * @return array Format: array(array('value' => '<value>', 'label' => '<label>'), ...)
+     */
+    public function toOptionArray(): array
+    {
+        return [
+            ['value' => 'bancontact', 'label' => __('Bancontact')],
```

Comment:
Should the values be constants as the terms are used in other files too

Reply:
Good point, I'll look at making them constants. Most probably in the config model?